recipes/llnl-sina/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "llnl-sina" %}
+{% set version = "1.9.1" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/LLNL/Sina/archive/{{ version }}.tar.gz
+    sha256: 28ca48c89b8e8280b41458f8a8eabf51408c68bc3a45a433258eb548b5447e8e
+
+build:
+  number: 0
+  noarch: python
+
+ 
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - sqlalchemy
+  run:
+    - python
+    - six
+    - sqlalchemy
+    - enum34
+
+test:
+  commands:
+    - python -c "import sina"
```

Comment:
Simplify the test phase with an import test instead of a command test that calls an import and add a `pip check` test.

```yaml
test
  commands:
    - python -c "import sina"
```

```yaml
test:
  imports:
    - sina
  commands:
    - pip check
  requires:
    - pip
```